src/site/facilities/service_location.drupal.liquid

```diff
@@ -101,6 +106,28 @@
         {% endfor %}
       {% endif %}
     </div>
+  {% else %}
+
+    {% comment %}
+      if fieldPhoneNumbersParagraph is empty and
+      fieldUseMainFacilityPhone is true use mainFacilityPhoneNumber
+    {% endcomment %}
+
+    {% assign useMainFacilityPhone  = locationEntity.fieldServiceLocation.0.entity.fieldUseMainFacilityPhone  %}
+    {% if useMainFacilityPhone %}
+      <div class="vads-u-display--flex vads-u-flex-direction--column" data-template="paragraphs/service_location">
+        {% if haveLocationName %}
+          <h4 class="force-small-header">Phone</h4>
+        {% else %}
+          <h5>Phone</h5>
+          <a aria-label="{{ mainFacilityPhoneNumber | accessibleNumber }}"
```

Comment:
Let's use the `fieldPhoneNumber` field here instead. We might need to pass it down from the parent, or assign it.

Reply:
ok